sycl/doc/extensions/experimental/sycl_ext_oneapi_device_global.asciidoc

```diff
@@ -1039,6 +1039,12 @@ the variable _dest_, the implementation throws an `exception` with the
 If `PropertyListT` contains the `device_image_scope` property and the _dest_
 variable exists in more than one _device image_ for this queue's device, the
 implementation throws an `exception` with the `errc::invalid` error code.
+
+If `PropertyListT` contains the `device_image_scope` property, kernels in the
+_device image_ containing the _dest_ variable must be compatible with the 
+device that is associated with the `handler`. If this is not the case, the 
+implementation throws an `exception` with the `errc::kernel_not_supported` 
+error code.
```

Comment:
@gmlueck I don't like the term `compatible` here. It is not clear that we are going to throw an error if there are no kernels that use the device_global.

Reply:
This wording that I proposed about "compatible" was intended to address this concern that you raised:

> It is undefined behavior to copy to or from a device global on a queue whose device cannot load a _device image_ that references the device global

I.e. "cannot load" --> "not compatible"

The request to throw an error when no kernel references the device global is something new that we just decided in the meeting today.  We would need some new wording to specify that error.

Reply:
Would this error be a runtime error? I was speaking with Artem today and it came up that there would be instances where we don't know in all cases if a particular device_image is going to be loaded or not, which would make this difficult to implement as a compile time error.

Reply:
@justin-rosner yes this will be a runtime error.